workers/unity/Assets/Playground/Scripts/EntityInitialization/GameObjectInitializationSystem.cs

```diff
@@ -32,50 +32,68 @@ public struct RemovedEntitiesData
             public SubtractiveComponent<GameObjectReference> NoGameObjectReference;
         }
 
+        public struct WorkerData
```

Comment:
It might be worth pulling this out into core given that its use everywhere as a common data struct.

Then no need to redefine in every system